RoadPosition road_position
InertialPosition nearest_position
float64 distance
